coremltools/converters/mil/mil/ops/defs/normalization.py

```diff
@@ -9,10 +9,15 @@
 class batch_norm(Operation):
     """
     Normalize input tensor ``x`` by ``mean`` and ``variance``, and optionally apply a
-    scale ``gamma`` and an offset ``beta``. ``mean``, ``variance``, ``gamma``, and ``beta``
-    must be 1-D tensors whose lengths are equal to the second axis ("depth" or "channel"
-    dimension) of ``x``.
-
+    scale ``gamma`` and an offset ``beta``:
+    
+    .. math::
+       y_i = \\gamma_i \\dfrac{ (x_i - mean_i)}{\\sqrt{variance_i^2 + epsilon}} + beta_i \\;,\\;i=1,....,C
```

Comment:
This should be "variance" instead of "variance^2"

Reply:
OK, changing `variance_i^2` to `variance`. Resulting formula (without escape backslashes): 

`y_i = \gamma_i \dfrac{ (x_i - mean_i)}{\sqrt{variance + epsilon}} + beta_i \;,\;i=1,....,C`